rtengine/dcraw.cc

```diff
@@ -10133,8 +10153,12 @@ void CLASS identify()
         width = raw_width = 6016;
         height = raw_height = 4014;
     } else if (!strcmp(model, "X-Pro3") || !strcmp(model, "X-T3") || !strcmp(model, "X-T30") || !strcmp(model, "X-T4") || !strcmp(model, "X100V") || !strcmp(model, "X-S10")) {
-        width = raw_width = 6384;
-        height = raw_height = 4182;
+        constexpr std::uint_fast16_t x_width = 6384, x_height = 4182;                                                               // RT
+        is_cropped = read_crop.complete && (x_width - raw_width > is_cropped_margin || x_height - raw_height > is_cropped_margin);  // RT
```

Comment:
Perhaps determine this from the crop mode (tag 0x104d, [reference](https://exiftool.org/TagNames/FujiFilm.html))?

Reply:
Thanks! I didn't know of this flag.